demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -0,0 +1,841 @@
+            click.secho('\nThe files are valid', fg='green')
+            return 0
+        else:
+            all_failing_files = '\n'.join(FOUND_FILES_AND_ERRORS)
+            self.print_ignored_files_report()
+            self.print_ignored_errors_report()
+            click.secho(f"\n=========== Found errors in the following files ===========\n\n{all_failing_files}\n",
+                        fg="bright_red")
+            click.secho('The files were found as invalid, the exact error message can be located above',
+                        fg='red')
+            return 1
+
+    def run_validation(self):
+        """Initiates validation in accordance with mode (i,g,a)
+        """
+        if self.validate_all:
+            return self.print_final_report(self.run_validation_on_all_packs())
+        if self.file_path:
+            return self.print_final_report(self.run_validation_on_specific_files())
+        if self.use_git:
+            return self.print_final_report(self.run_validation_using_git())
+
+    def run_validation_on_specific_files(self):
+        """Run validations only on specific files
+        """
+        valid_files = set()
+
+        for path in self.file_path.split(','):
+            error_ignore_list = self.get_error_ignore_list(get_pack_name(path))
+
+            if os.path.isfile(path):
+                click.secho('\n================= Validating file =================', fg="bright_cyan")
+                valid_files.add(self.run_validations_on_file(path, error_ignore_list))
+
+            else:
+                path = path.rstrip('/')
+                dir_name = os.path.basename(path)
+                if dir_name in CONTENT_ENTITIES_DIRS:
+                    click.secho(f'\n================= Validating content directory {path} =================',
+                                fg="bright_cyan")
+                    valid_files.add(self.run_validation_on_content_entities(path, error_ignore_list))
+                else:
+                    rest_of_path = path.replace(f'/{dir_name}', '')
+                    if os.path.basename(rest_of_path) == PACKS_DIR:
+                        click.secho(f'\n================= Validating pack {path} =================',
+                                    fg="bright_cyan")
+                        valid_files.add(self.run_validations_on_pack(path))
+
+                    else:
+                        click.secho(f'\n================= Validating package {path} =================',
+                                    fg="bright_cyan")
+                        valid_files.add(self.run_validation_on_package(path, error_ignore_list))
+
+        return all(valid_files)
+
+    def run_validation_on_all_packs(self):
+        """Runs validations on all files in all packs in repo (a)
+
+        Returns:
+            bool. true if all files are valid, false otherwise.
+        """
+        click.secho('\n================= Validating all files =================', fg="bright_cyan")
```

Comment:
use const please

Reply:
this will be the same, you will just find it elsewhere and will jump to it's usage